spec/features/shelters/user_can_see_all_shelters_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe "shelters index page", type: :feature do
+  it "can see all names of the shelters in system" do
+    shelter_1 = Shelter.create(name: "Dumb Friends League")
+    shelter_2 = Shelter.create(name: "Pick a Pal")
+
+    visit "/shelters"
+
+    expect(page).to have_content(shelter_1.name)
```

Comment:
:id will have all the "info" regarding the shelter. Will .name being duplicated cause any problems?